scaleapi/__init__.py

```diff
@@ -64,18 +64,22 @@ def get_task(self, task_id: str) -> Task:
         endpoint = f"task/{task_id}"
         return Task(self.api.get_request(endpoint), self)
 
-    def cancel_task(self, task_id: str) -> Task:
+    def cancel_task(self, task_id: str, clear_unique_id: bool) -> Task:
```

Comment:
We can default it to `None` or `false` so it will still be an optional parameter for existing users and won't break their usage.